src/planscape/rscripts/forsys.R

```diff
@@ -10,9 +10,12 @@ library("glue")
 library("forsys")
 library("sf")
 library("dplyr")
+library("purrr")
 
 readRenviron("src/planscape/planscape/.env")
 
+COST_PER_ACRE <- 2470
```

Comment:
We're still eventually allowing users to configure this, correct?  Can you leave a TODO to change this later?

Reply:
sg